tests/prototest/test.proto

```diff
@@ -40,7 +40,7 @@ message ProtoMessage {
   }
   optional int32 c = 12 [default = 16];
   optional int64 d = 1 [default = 0];
-  optional uint32 p = 1;
+  optional uint32 p = 40;
```

Comment:
Add proto with problems such as duplicate id, use id from reserved, ... and add to tests